components/automate-grpc/protoc-gen-policy/testdata/imported_method_arguments.proto

```diff
@@ -3,13 +3,13 @@ package chef.automate.domain.imported_method_arguments;
 option go_package = "github.com/chef/automate/components/mock-service/mock";
 
 import "google/api/annotations.proto";
-import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
+import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/testdata/messages.proto";
 
 service Mock {
   rpc MockMethod (messages.Foo) returns (messages.Bar) {
     option (google.api.http).get = "/auth/foo/baz";
-    option (chef.automate.api.policy).resource = "mock:foo:{bar}";
-    option (chef.automate.api.policy).action = "action";
+    option (chef.automate.api.iam.policy).resource = "mock:foo:{bar}";
```

Comment:
how/where is this example proto used?

Reply:
Everything under `testdata` is used in the unit tests; I noted that on L43 here: https://github.com/chef/automate/pull/3050/files#diff-441ca6e3b50b8e2957783406e6b1f944R43